### PR TITLE
Update configure-tls.md

### PR DESCRIPTION
**Pull request by dongluochen**
Signed-off-by: Dong Chen <dongluo.chen@docker.com>

### CODE REVIEW

**Comment by dongluochen** [comment]
I think there is one more miss on IP
```
$ echo subjectAltName = IP:10.10.10.20,IP:127.0.0.1 > extfile.cnf

$ openssl x509 -req -days 365 -sha256 -in server.csr -CA ca.pem -CAkey ca-key.pem \
  -CAcreateserial -out server-cert.pem -extfile extfile.cnf
```

**Comment by dongluochen** [comment]
cc @moxiegirl.

**Comment by moxiegirl** [comment]
The content LGTM @dongluochen  --- need to get your last technical question answered.  We'll need to cherry pick this.

**Comment by moxiegirl** [comment]
Assign it to me after the merge and I'll cherry pick

**Comment by vieux** [comment]
@dongluochen @moxiegirl I added a commit to remove the docker build step and use -v instead

**Comment by dongluochen** [comment]
LGTM.

File: docs/configure-tls.md [hunk 1]
```diff
@@ -405,57 +406,27 @@ discovery backend uses Docker Hub and is not recommended for production use.
...
     keys.
 
-7. Run a `docker ps` command to verify that your Swarm manager container is up
+2. Run a `docker ps` command to verify that your Swarm manager container is up
 and running.
 
         $ docker ps
         CONTAINER ID   IMAGE               COMMAND                  CREATED          STATUS          PORTS                              NAMES
-        035dbf57b26e   nigel/swarm-tls     "/swarm manage --tlsv"   7 seconds ago    Up 7 seconds    2375/tcp, 0.0.0.0:3376->2376/tcp   compassionate_lovelace
+        035dbf57b26e   swarm               "/swarm manage --tlsv"   7 seconds ago    Up 7 seconds    2375/tcp, 0.0.0.0:3376->3376/tcp   compassionate_lovelace
```
Comment (nishanttotla) [inline docs/configure-tls.md]: Small question, what is `2375/tcp`? Would that change to `3376`?

Comment (vieux) [inline docs/configure-tls.md]: `2375` is exposed in the image that's why it's showed as `2375/tcp` in `docker ps`

**Comment by nishanttotla** [comment]
LGTM

**Comment by moxiegirl** [comment]
lgtm

**Pull request merged**